erp_integration/erp_staged_records.md

```diff
@@ -11,6 +11,28 @@ section_title: ERP Integration
 For an ERP object to be imported into Procore, records must be staged in the ERP Integration tool where an accountant can review and approve records for import. 
 Integrators can use the publicly accessible [**Staged Records API**](https://developers.procore.com/reference/rest/v1/erp-staged-record).
 
+## Configuration
```

Comment:
Just adding this because it came up in some slack messages. We should let integrators know that metadata configs are required for Staged Records